src/docfx/build/redirection/RedirectionProvider.cs

```diff
@@ -225,5 +221,24 @@ private static string RemoveTrailingIndex(string redirectionUrl)
             var (url, _, _) = UrlUtility.SplitUrl(redirectionUrl);
             return url.EndsWith("/index", PathUtility.PathComparison) ? url.Substring(0, url.Length - "index".Length) : url;
         }
+
+        private static string RemoveLeadingLocale(string redirectionUrl, string hostName)
+        {
+            var (redirectHostName, redirectPath) = UrlUtility.SplitBaseUrl(redirectionUrl);
+            if (!string.Equals(redirectHostName, hostName, StringComparison.OrdinalIgnoreCase))
+            {
+                return redirectionUrl;
+            }
+
+            if (redirectPath.IndexOf('/') is int slashIndex && slashIndex < 0)
```

Comment:
isn't `redirectPath.IndexOf('/')` always `int`?

Reply:
'slashIndex' is used later, and IndexOf doesn't have a 'out int index' overload.